content/en/_index.md

```diff
@@ -36,99 +36,99 @@ section2:
 
 section3:
   title: Benefits to different teams
-  content: Multi-tenancy makes different teams to securely deploy and maintain containerized applications from the cloud to the edge.  It allows developers to deploy code via several clicks on intuitive console, brings centralized observability and powerful DevOps strategy for Ops team, helps Infra team to install and maintain Kubernetes cluster with flexible network and solution solutions, avoid locking team in to a single vendor eco-system.
+  content: Multi-tenancy enables different teams to securely deploy and maintain containerized applications from the cloud to the edge.  It allows developers to deploy code via several clicks on intuitive consoles, brings centralized observability and powerful DevOps strategies for the Ops team, helps the Infra team to install and maintain Kubernetes cluster with flexible network and efficient solutions, and avoids locking teams into a single-vendor ecosystem.
   children:
     - name: Infra Team
       content: Automated installation, scaling, and upgrades from cloud to data center
       icon: /images/home/7.svg
       children:
         - content: Improve your environment utilization and reduce internal infrastructure costs
         - content: Support multi-cluster and multi-cloud Kubernetes management, avoiding vendor lock-in
-        - content: Provides security enhancements, supports multiple storage and network solutions
-        - content: Fully trusted, delivers a certified Kubernetes platform and distribution
+        - content: Provide security enhancements, multiple storage and network solutions
+        - content: Deliver a trustworthy and certified Kubernetes platform and distribution
 
     - name: Developers
-      content: Focus on your bussiness, others are run in automated tools
+      content: Focus on your business as automation tools do the rest
       icon: /images/home/74.png
       children:
-        - content: Smooth user experience, reduce the learning curve of the cloud native stack
+        - content: Create a smooth user experience and reduce the learning curve of the cloud native stack
```

Comment:
`Create a smooth user experience` looks strange to me. I think there is no need `Create a`.

Reply:
This is for format uniformity for descriptions under "Benefits to different teams" (v+n)